server.js

```diff
@@ -0,0 +1,7 @@
+const app = require('./functions/authorization/authorization')
+
+const port = 3000;
```

Comment:
```suggestion
const PORT = 3000;
```
As a practice let's use the "upper snake case" to define constants that we define in code.